examples/react-signer/src/hook/use-signer.ts

```diff
@@ -15,12 +15,19 @@ export enum State {
   PROCESSING,
 }
 
+;(async () => {
```

Comment:
seems better not to separate from the hook and move exactly this code to useEffect if it works this way

Reply:
Fixed, thank you!